AutomatedTesting/Registry/asset_cache_server_settings.setreg

```diff
@@ -0,0 +1,32 @@
+{
+    "O3DE": {
+        "AssetProcessor": {
+            "Settings": {
+                "Server": {
+                    "cacheServerAddress": "",
+                    "assetCacheServerMode": "inactive",
+                    "ACS Atom Image Builder": {
```

Comment:
Minor: did we consider having these in a clear deliminted container ie AssetSettings: [] so its clear what these are and to make the "envelope" part easier to read?